packages/react/src/components/withClerk.tsx

```diff
@@ -48,5 +45,5 @@ export const WithClerk: React.FC<{
     return null;
   }
 
-  return <LoadedGuarantee>{children(clerk as unknown as LoadedClerk)}</LoadedGuarantee>;
```

Comment:
❓ Can you please elaborate what replaces the loaded guarantee?

Reply:
Nothing used the LoadedGuarantee (or "structure context") since the v4 hook change. We kept it around in case we wanted to use it to throw better errors in cases where the Clerk hooks were used outside the Clerk provider but ended up doing that by simply checking wether the ClerkProvider context exists or not.

Extra context: The LoadedGuarantee (or "structure context") used to wrap all SignedIn and SignedOut components and the older hooks could only be used when nested under these components - this is no longer the case as all hooks return `isLoaded`

Reply:
cc @dimkl